features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/Configuration.jsx

```diff
@@ -190,6 +199,23 @@ class Configuration extends React.Component {
             return apiTransport.includes(type);
         }
     }
+    setSecurityScheme = (updatedSecurityScheme) => {
+        this.setState((oldState) => {
+            let { securityScheme } = oldState;
+            securityScheme = updatedSecurityScheme;
+            return { securityScheme };
+        });
+    }
+    addToArray(element, array) {
+        if (!array.includes(element)) {
+            array.push(element);
+        }
+    }
+    removeFromArray(element, array) {
+        if (array.includes(element)) {
+            array.splice(array.indexOf(element), 1);
```

Comment:
this also we can move this method in to APISecurity component right?

Reply:
it is used within configuration class as well